pkg/log/ansHook.go

```diff
@@ -0,0 +1,109 @@
+package log
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/SAP/jenkins-library/pkg/ans"
+	"github.com/SAP/jenkins-library/pkg/xsuaa"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"strings"
+)
+
+// ANSHook is used to set the hook features for the logrus hook
+type ANSHook struct {
+	client ans.Client
+	event  ans.Event
+}
+
+// NewANSHook creates a new ANS hook for logrus
+func NewANSHook(config ans.Configuration, correlationID string) ANSHook {
```

Comment:
Manually it is difficult to switch it off. But it might be good to switch it off automatically.
@mkunev, @dzhurdzhina, @simeonPetkov96 is there a way to test the service and service key?